core/src/saros/account/XMPPAccountStore.java

```diff
@@ -489,16 +482,34 @@ public void changeAccountData(
     notifyAccountStoreListeners();
   }
 
+  /**
+   * Returns the current default account.
+   *
+   * @return the default account or <code>null</code> if the default account is not set
+   */
+  public synchronized XMPPAccount getDefaultAccount() {
+    return defaultAccount;
+  }
+
   /**
    * Returns the current active account.
    *
    * @return the active account
    * @throws IllegalStateException if the account store is empty
+   * @deprecated Will be removed soon, do not use
```

Comment:
Refer to `getDefaultAccount()` as the new alternative?
```suggestion
   * @deprecated Will be removed soon, use {@link getDefaultAccount} instead
```